cli/exchange/node.go

```diff
@@ -180,6 +185,10 @@ func NodeCreate(org, nodeIdTok, node, token, userPw, arch string, nodeName strin
 		} else {
 			msgPrinter.Printf("Node %v created.", nodeId)
 			msgPrinter.Println()
+			if strings.Contains(resp.Msg, "Warning: you are within") {
```

Comment:
The message gotten from the exchange might be translated if the exchange has a non-English locale set. Comparing  the text here will have problem in this case.